pavelib/assets.py

```diff
@@ -41,10 +41,10 @@
 
 # Common lookup paths that are added to the lookup paths for all sass compilations
 COMMON_LOOKUP_PATHS = [
-    path("common/static"),
-    path("common/static/sass"),
+    path('common/static'),
+    path('common/static/sass'),
     path('node_modules'),
-    path('node_modules/edx-pattern-library/node_modules'),
```

Comment:
@andy-armstrong am I okay to delete this line as part of the change?

Reply:
Sorry, I should have mentioned this one. Yes, this is no longer necessary. It was used for earlier versions of npm which installed dependencies into subdirectories, but now they are all shared at the root of node_modules.

Reply:
No worries, thanks!